tests/utils/utils.go

```diff
@@ -332,3 +332,26 @@ func calculateTxParams(ctx context.Context, wsClient ethclient.Client, fundedAdd
 
 	return gasFeeCap, gasTipCap, nonce
 }
+
+func createNativeTransferTransaction(
+	ctx context.Context,
+	network SubnetTestInfo,
+	fromAddress common.Address,
+	fromKey *ecdsa.PrivateKey,
+	recipient common.Address,
+	amount *big.Int,
+) *types.Transaction {
+	gasFeeCap, gasTipCap, nonce := calculateTxParams(ctx, network.ChainWSClient, fundedAddress)
+
+	tx := types.NewTx(&types.DynamicFeeTx{
+		ChainID:   network.ChainIDInt,
+		Nonce:     nonce,
+		To:        &recipient,
+		Gas:       DefaultTeleporterTransactionGas,
```

Comment:
do we want this utils function to always pass in gas for the Teleporter transaction?

Reply:
Updated to be 21,000 since it's always the gas used by a native transfer